components/raftstore-v2/src/operation/command/admin/split.rs

```diff
@@ -610,7 +797,9 @@ mod test {
         // Split will create checkpoint tablet, so if there are some writes before
         // split, they should be flushed immediately.
         apply.apply_put(CF_DEFAULT, b"k04", b"v4").unwrap();
-        assert!(!apply.write_batch_mut().as_ref().unwrap().is_empty());
+        assert!(!WriteBatch::is_empty(
```

Comment:
Why?

Reply:
Two traits imported include is_empty, so use this way to let compiler happy.